lib/decorators-callbacks.js

```diff
@@ -18,11 +18,24 @@ module.exports.decorateLambdaReqCallback = function(getPrincipalId) {
   };
 };
 
-module.exports.decorateAddCORSCallback = function() {
+module.exports.decorateAddCORSCallback = function(corsOpts, isOptionMethod) {
+  // TODO: get default cors options from variables
+  const cors = {
+    origins: ['*'],
+    methods: ['GET', 'PUT', 'HEAD', 'PATCH', 'POST', 'DELETE', 'OPTIONS'],
+    headers: ['Authorization', 'Content-Type', 'x-amz-date', 'x-amz-security-token']
+  };
+  if(corsOpts instanceof Object) {
+    Object.assign(cors, corsOpts);
```

Comment:
to make a new object (if we move `cors` outside the scope) we can: `Object.assign({}, cors, corsOpts)`